CMSgov/design-system
# [WNMGDS-1693] Remove banner role from Dialog.

## Summary

[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1693)

Remove `banner` role from Dialog header. This was unnecessary as header already has that role and the extra role was breaking accessibility testing for some team.

## How to test

1. Pull branch
2. Run `yarn storybook`
3. Visit Dialog story and see no change in behavior

### If this is a change to code:


### If this is a change to documentation:
